bundle/src/test/resources/com/adobe/acs/commons/workflow/process/impl/CopyPropertiesProcessTest.json

```
@@ -0,0 +1,35 @@
+{
+  "one.png" : {
```

You should name the test resources something that correlates to the test you are validating - so that it is clear what is impacted by changing the test data.
```suggestion
  "copyProperties.png" : {
```